quill-jdbc-zio/src/main/scala/io/getquill/context/qzio/ZioJdbcUnderlyingContext.scala

```diff
@@ -118,19 +117,6 @@ abstract class ZioJdbcUnderlyingContext[+Dialect <: SqlIdiom, +Naming <: NamingS
              .refineToOrDie[E]
     } yield r
 
-  private[getquill] def streamWithoutAutoCommit[A](
-    f: ZStream[Connection, Throwable, A]
-  ): ZStream[Connection, Throwable, A] =
-    for {
-      conn          <- ZStream.service[Connection]
-      autoCommitPrev = conn.getAutoCommit
-      r <- ZStream
-             .acquireReleaseWith(ZIO.attempt(conn.setAutoCommit(false))) { _ =>
-               ZIO.succeed(conn.setAutoCommit(autoCommitPrev))
```

Comment:
The test that was failing is testing that this second `conn.setAutoCommit` call (the "release" call), when failing, wasn't preventing the `connection.close()` call from happening.
I have no idea how this test could have passed. Because this call when failing just "crash" the IO execution and there's no where in Quill code (and it's a good thing) a piece of code that tries to recover from this "crash" to ensure that the connection is closed.

So basically, this tests was most probably passing while it most probably shouldn't have passed